### PR TITLE
Training with subsets to master

### PR DESCRIPTION
**Pull request by Kamil-Kisielewicz**

### CODE REVIEW

File: spit/classifier.py [hunk 1]
```diff
@@ -141,6 +142,152 @@ def compare_with_best(self, test_dset, test_labels, file_path):
...
+    :param spit_path & param save_path:
+      Path to the spit images and where the model will be saved respectively.
+      ***Environmental variables must be set by caller or the path must be passed manually.***
+
+    Returns:
+    :returns history:
+      Tensorflow History object containing loss and accuracy data over the training.
+
+    """
+    # choose the model
```
Comment (profxj) [inline spit/classifier.py]: While at first it made some sense to me to have this be a method under `Classifier`, these lines make it obvious that we want to use this code on any number of models.  Therefore, I think we should make this a stand-alone method in the `train.py` module.

Comment (profxj) [reply spit/classifier.py]: Further, it isn't as if we access the model internals (i.e. `self`) in any sigfnificant way in the remainder of this method

Comment (profxj) [reply spit/classifier.py]: We can and should make `split_array` a method of `train.py` as well.

**Review by profxj — CHANGES_REQUESTED** [review]
This looks great.  My only substantial request is that the code
be moved from classifier.py into train.py.   Makes more sense 
to me to have it be distinct from the model.

File: spit/classifier.py (hunk 1 above)
Comment (Kamil-Kisielewicz) [reply spit/classifier.py]: Agreed, moved and will submit new request.

**Comment by profxj** [comment]
Only one last request from me.  Can you name the method something other than `_train`?
More to the point, when you start with `_` it is hidden from easy view.  You could call it
`standard_train` or `keras_train` or something like that.  Thanks!

**Comment by profxj** [comment]
merging

**Pull request merged**